traits/ctraits.c

```diff
@@ -4908,6 +4908,46 @@ PyMODINIT_FUNC PyInit_ctraits(void) {
                          (PyObject *) &trait_type ) < 0 )
        return NULL;
 
+    /* Default value type constants */
+    if ( PyModule_AddIntConstant(
```

Comment:
I'm not especially keen on the assign-and-test style, in general. I think it would be more readable to separate the assignment from the error check. Also, let's follow PEP 7 in new code, and always use braces, even for single-statement blocks.

```
err  = PyModule_AddIntConstant(module, "_CONSTANT_DEFAULT_VALUE", CONSTANT_DEFAULT_VALUE);
if (err) {
    return NULL;
}
```